manifests/n/NordVPN/NordVPN/7.2.6.0/NordVPN.NordVPN.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: NordVPN.NordVPN
+PackageVersion: 7.2.6.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
From the validation error, it appears a 64-bit OS is required. 
![image](https://user-images.githubusercontent.com/114952565/207120088-f7f6a69d-413d-4b65-8ba0-3d6e99dc28e4.png)